src/Collection/index.js

```diff
@@ -83,6 +83,10 @@ export default class Collection<Record: Model> {
     return new Query(this, clauses)
   }
 
+  queryWithDeleted(...clauses: Clause[]): Query<Record> {
```

Comment:
what legitimate use cases are there for fetching deleted records?

Reply:
@radex undelete?!